index.js

```diff
@@ -113,7 +113,11 @@ exports.fromMapFileComment = function (comment, dir) {
 
 // Finds last sourcemap comment in file or returns null if none was found
 exports.fromSource = function (content, largeSource) {
-  if (largeSource) return convertFromLargeSource(content);
+  if (largeSource) {
+    var res = convertFromLargeSource(content);
+    if (typeof res === 'undefined') res = null;
+    return res;
```

Comment:
I was talking about just the last two.

Just do:

```js
if (largeSource) {
  var res = convertFromLargeSource(content);
  if (!res) return null;
  return res;
}
```
Don't _only_ check for `undefined` cause you also wanna return `null` if it's null. Even though that would happen anyways on the next line, but that is convoluted. Checking for falsy is clearer IMO.